packages/core/database/lib/index.js

```diff
@@ -33,7 +33,13 @@ class Database {
 
     this.schema = createSchemaProvider(this);
 
-    this.migrations = createMigrationsProvider(this);
+    if (config.connection.disableMigrations) {
+      this.migrations = {
+        shouldRun: () => Promise.resolve(false),
+      };
+    } else {
+      this.migrations = createMigrationsProvider(this);
```

Comment:
this isn't in the right place in the config it should be part of settings instead. you will need to merge the incoming settings with the one on line 19 however. 

Also this condition should be handled in the migrations Provider shouldRUn function directly

Reply:
thanks for the review @alexandrebodin 
I've updated as advised, please let me know